pkg/3scale/amp/operator/apicast_options_provider.go

```diff
@@ -440,6 +441,11 @@ func (a *ApicastOptionsProvider) additionalPodAnnotations() map[string]string {
 	annotations := map[string]string{
 		APIcastEnvironmentCMAnnotation: a.envConfigMapHash(),
 	}
+	if a.apimanager.Spec.Apicast.StagingSpec.Annotations != nil {
+		for k, v := range a.apimanager.Spec.Apicast.StagingSpec.Annotations {
+			annotations[k] = v
+		}
+	}
```

Comment:
do we need this  and if so why is it only for staging?

Reply:
removed. It was a bug. Thank you very much!
Changes - in separate commit.